### PR TITLE
global: Rename sd_altpal_file to altpal_file

### PR DESCRIPTION
**Pull request by mrannanj**

### CODE REVIEW

File: include/formats/altpal.h [hunk 1]
```diff
@@ -9,25 +9,25 @@
 
 #include "formats/palette.h"
 
-#ifndef _SD_ALTPAL_H
-#define _SD_ALTPAL_H
+#ifndef _ALTPAL_H
```
Comment (turol) [inline include/formats/altpal.h:12]: This is illegal. Identifiers starting with _ and a capital letter are reserved for the C implementation. Use `ALTPAL_H`. If there are other such guards you should fix them too.

Comment (mrannanj) [reply include/formats/altpal.h:12]: Starting with underscore is the convention in this codebase for include guards, need to ask @katajakasa

Comment (katajakasa) [reply include/formats/altpal.h:12]: Well, to be fair, using them is totally legan, just discouraged. Compiler would complain loudly if it was illegal. But sure, I don't mind, might as well change it.

Comment (turol) [reply include/formats/altpal.h:12]: With the right command line option clang does complain about them.

Comment (katajakasa) [reply include/formats/altpal.h:12]: That is not complaining /loudly/ :) But I'm not going to argue further when I essentially agree with you :P

Comment (mrannanj) [reply include/formats/altpal.h]: Added a commit to remove the underscores.

**Pull request merged**